pkg/storegateway/bucket_stores.go

```diff
@@ -299,25 +299,22 @@ func (u *BucketStores) Series(req *storepb.SeriesRequest, srv storepb.Store_Seri
 	}
 
 	store := u.getStore(userID)
+	userBkt := bucket.NewUserBucketClient(userID, u.bucket, u.limits)
```

Comment:
We are initializing a new bkt client every `Series` request? Can we just use `u.bucket` here?

Reply:
I had to do this to unwrap the error, but creating this here is just creating an object (it does not do anything).